components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -398,31 +386,33 @@ public void updateTierPermissions(String tierName, String permissionType, String
      * @return   Set of Documents and APIs
      * @throws APIManagementException
      */
-    public Map<Documentation, API> searchAPIsByDoc(String searchTerm, String searchType) throws APIManagementException;
-    
+    public Map<Documentation, API> searchAPIsByDoc(String searchTerm, String searchType) throws APIManagementException;   
```

Comment:
Method comment updated with more details in the commit af71bc61fe613513fd32a4d6243a7a3406c5bd44